UefiTestingPkg/AuditTests/PagingAudit/UEFI/X64/PagingAuditProcessor.c

```diff
@@ -666,28 +547,22 @@ GetFlatPageTableData (
                 }
               }
 
-              // Increase the count.
-              // If we have room for more Page Table entries, add one.
               My4KCount++;
               if (My4KCount <= *Pte4KCount) {
-                Pte4KEntries[My4KCount-1] = Pte4K[Index1].Uint64;
+                Pte4KEntries[My4KCount-1] = Pte4K[Index1];

```

Comment:
We don't need the `.Uint64` any longer?

Reply:
Uint64 was a part of a union and allowed the capture of the whole 64 bits of the page. Because the bitfields and unions were removed, we don't need this anymore.